sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -1061,6 +1079,12 @@ def test_global_forecasting_series(self, estimator_instance, n_columns):
         assert set(y_test.index.names).issubset(y_pred.index.names)
 
     def test_global_forecasting_no_X(self, estimator_instance):
+        global_forecast = estimator_instance.get_tag(
```

Comment:
These checks are the same for the four tests in the file? Can you extract it in a new method?

Reply:
It's extracted to a method.